### PR TITLE
Database set up

### LINKED ISSUE #4: Create SQL Schema Script
As a developer, when I run this SQL script, the correct tables should be created in the database, so that I can fill them.

This will be the Schema.sql script that creates the tables in the RDS (short term database).

### PR DESCRIPTION
**Pull request by CalebMathers**
Fixes #4 , #5 , #6 , and #7 

Wrote DDL SQL to create all the tables for our schema, wrote insertions for all the static data regarding plants, locations, and botanists. Made two shell scripts; one to connect to an interactive session on the RDS and one to connect and reset the DB.

### CODE REVIEW

File: seeding/reset.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+source .env
+sqlcmd -S $DB_HOST,$DB_PORT -U $DB_USER -P $DB_PASSWORD -d $DB_NAME -i 'schema.sql'
```
Comment (megansam5) [inline seeding/reset.sh:2]: Nice script for easy access, cood that you have two as well!

File: seeding/schema.sql [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+(39, 'Blanket Flower', 'Gaillardia', 37, 1),
+(40, 'Amaryllis', 'Hippeastrum', 38, 3),
+(41, 'Angel Wings', 'Caladium Bicolor', 39, 1),
+(42, 'Spider Plant', 'Chlorophytum Comosum', 40, 1),
+(44, 'Norfolk Island Pine', 'Araucaria Heterophylla', 41, 2),
+(45, 'Wax Begonia', 'Begonia Semperflorens', 2, 2),
+(46, 'Pink Lantern', 'Medinilla Magnifica', 42, 3),
+(47, 'Red Powder Puff', 'Calliandra Haematocephala', 43, 3),
+(48, 'ZZ Plant', 'Zamioculcas Zamiifolia', 44, 1),
+(49, 'Jade Plant', 'Crassula Ovata', 45, 3),
```
Comment (megansam5) [inline seeding/schema.sql:145]: ngl I'm very impressed that you managed to find all this info, well done!

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+venv/
+.env
```
[the remark this answers is not in the document]
Comment (MurasatoNaoya) [reply .gitignore:2]: Noice. Don't want to be uploading our secrets, do we.

File: seeding/connect.sh
Comment (MurasatoNaoya) [inline seeding/connect.sh:1]: Connects to the database in an automated way, saves a lot of typing. Nice contribution!

File: seeding/reset.sh
Comment (MurasatoNaoya) [inline seeding/reset.sh:1]: Necessary script for when a wipe is necessary, God forbid.

File: seeding/schema.sql (hunk 2 above)
Comment (MurasatoNaoya) [inline seeding/schema.sql:145]: Never doubt a man on a mission, background music, and the Internet, apparently. Great to have this, as we can account for edge cases where certain entries don't contain a plant's common name,  only their scientific name; and vice versa. Will save us a lot of headache later if we ever have to retrace insertions. Nice for completeness too.

**Pull request merged**